homeassistant/components/enigma2/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""Support for Enigma2 devices."""
+from homeassistant.components.discovery import SERVICE_ENIGMA2
+from homeassistant.helpers.discovery import load_platform
+from homeassistant.helpers import discovery
+
+DOMAIN = 'enigma2'
+
+DEPENDENCIES = ['http']
```

Comment:
Is this necessary?

Reply:
Good question. I stole this line from a number of other discovery methods which rely on the standard http discovery.

Reply:
I've removed that now.